src/rocketmq_client.erl

```diff
@@ -68,33 +68,41 @@ get_status(Pid) ->
 %%--------------------------------------------------------------------
 init([Servers, Opts]) ->
     State = #state{servers = Servers, opts = Opts},
-    case get_sock(Servers, undefined) of
+    SSLOpts = maps:get(ssl, Opts, undefined),
```

Comment:
I noticed a few other libs use `ssl_opts` maybe we can follow the convention?

Reply:
Sure, it is better to be as consistent as possible. Will change to ssl_opts.

Reply:
Fixed